### PR TITLE
Add microservice to sync individual repos

### PR DESCRIPTION
**Pull request by sidpremkumar**
Relates to JIRA: PVHEALTH-272

### CODE REVIEW

**Comment by sidpremkumar** [comment]
Using Patternfly 4, our pages look like this: 
image

**Comment by sidpremkumar** [comment]
Success page looks like this, returns what repos were synced: 
image

File: openshift/sync2jira-sync-page-template.yaml [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+  value: "mock_confluence_space"
+- name: INITILIZE
+  displayName: True/False Initilize our repos on startup
+  value: "0"
+- name: IMAGE_URL
+  displayName: Image URL:TAG to pull from
+  value: "quay.io/redhat-aqe/sync2jira:sync-page"
+- name: JIRA_PNT_PASS
+  displayName: PNT password in base64
+  requiered: true
```
Comment (ralphbean) [inline openshift/sync2jira-sync-page-template.yaml:33]: Typo.  :)

File: openshift/sync2jira-sync-page-template.yaml [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+- name: JIRA_OMEGAPRIME_PASS
+  displayName: Omegaprime password in base64
+  requiered: true
```
Comment (ralphbean) [inline openshift/sync2jira-sync-page-template.yaml:36]: Typo.  :)

File: openshift/sync2jira-sync-page-template.yaml [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+- name: GITHUB_TOKEN
+  displayName: GitHub token in base64
+  requiered: true
```
Comment (ralphbean) [inline openshift/sync2jira-sync-page-template.yaml:39]: Typo.  :)

File: openshift/sync2jira-sync-page-template.yaml [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+- name: PAAS_DOMAIN
+  displayName: Domain to use for the service
+  requiered: true
```
Comment (ralphbean) [inline openshift/sync2jira-sync-page-template.yaml:42]: Typo.  :)

**Pull request merged**